File: sBuilding/Trunk/Client/sBuilding/sTimeKeeping/Model/SessionWorking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace sTimeKeeping.Model
{
   
    public class SessionWorking
    {
        public SessionWorking()
        {
            isCheckSession = 1 ; // default value
            hoursBegin = 0; // default value
            minuteBegin = 0; // default value
            hoursEnd = 24; // default value
            minuteEnd = 0; // default value
        }
        public int isCheckSession { get; set; }
        
        public int hoursBegin { get; set; }
       
        public int minuteBegin { get; set; }
      
        public int hoursEnd { get; set; }
       
        public int minuteEnd { get; set; }

    }
}

File: sBuilding/Trunk/Server/Java/timekeepingservice/src/com/swt/timekeeping/customer/object/SessionWorking.java
package com.swt.timekeeping.customer.object;

public class SessionWorking {
	 private int isCheckSession;
     
	 private int hoursBegin;
    
	 private int minuteBegin;
   
	 private int hoursEnd; 
    
	 private int minuteEnd;

	public SessionWorking(int isCheckSession, int hoursBegin, int minuteBegin,
			int hoursEnd, int minuteEnd) {
		super();
		this.isCheckSession = isCheckSession;
		this.hoursBegin = hoursBegin;
		this.minuteBegin = minuteBegin;
		this.hoursEnd = hoursEnd;
		this.minuteEnd = minuteEnd;
	}

	public int getIsCheckSession() {
		return isCheckSession;
	}

	public void setIsCheckSession(int isCheckSession) {
		this.isCheckSession = isCheckSession;
	}

	public int getHoursBegin() {
		return hoursBegin;
	}

	public void setHoursBegin(int hoursBegin) {
		this.hoursBegin = hoursBegin;
	}

	public int getMinuteBegin() {
		return minuteBegin;
	}

	public void setMinuteBegin(int minuteBegin) {
		this.minuteBegin = minuteBegin;
	}

	public int getHoursEnd() {
		return hoursEnd;
	}

	public void setHoursEnd(int hoursEnd) {
		this.hoursEnd = hoursEnd;
	}

	public int getMinuteEnd() {
		return minuteEnd;
	}

	public void setMinuteEnd(int minuteEnd) {
		this.minuteEnd = minuteEnd;
	} 
	 
	 
}
